.github/release-drafter.yml

```diff
@@ -0,0 +1,89 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+name-template: "v$RESOLVED_VERSION"
+tag-template: "v$RESOLVED_VERSION"
+change-template: "- #$NUMBER $TITLE @$AUTHOR"
+sort-direction: ascending
+#
+categories:
+  - title: "🚨 Breaking changes"
+    labels:
+      - "breaking-change"
+  - title: "✨ New features"
+    labels:
+      - "new-feature"
+  - title: "🐛 Bug fixes"
+    labels:
+      - "bugfix"
+  - title: "🚀 Enhancements"
+    labels:
+      - "enhancement"
+      - "refactor"
+      - "performance"
+  - title: "🧰 Maintenance"
+    labels:
+      - "maintenance"
+      - "ci"
+  - title: "📚 Documentation"
+    labels:
+      - "documentation"
+  - title: "⬆️ Dependency updates"
+    collapse-after: 5
+    labels:
+      - "dependencies"
```

Comment:
We're using a different set of standard labels at the moment but it could be adjust later on I imagne.

Reply:
Sorry, I didn't see your label. We can align, it's  better.

Reply:
Yeah, no problem. Labels are not created yet in this repo. It's much more a concern about aligning this with other repos. We'll check that in a second time.